src/api/data/cache/cache.go

```diff
@@ -148,37 +148,84 @@ func (c *cachedCharts) Refresh() error {
 		if err != nil {
 			return err
 		}
+
+		// 2 - Parse repo index
 		charts, err := helpers.ParseYAMLRepo(body, repo.Name)
 		if err != nil {
 			return err
 		}
+
+		// 3 - Process elements in index
 		var chartsWithData []*models.ChartPackage
+		// Buffered channel
+		ch := make(chan chanItem, len(charts))
```

Comment:
I thought that closing channels were only needed to be done from the sender to indicate the receiver from this fact.

According to this post (I also read it in A.A Donovan's book), leaving a channel open without senders will be automatically garbage collected. http://stackoverflow.com/questions/8593645/is-it-ok-to-leave-a-channel-open https://groups.google.com/forum/#!msg/golang-nuts/pZwdYRGxCIk/qpbHxRRPJdUJ

It am ok closing it, I want to know if in your opinion that's a best practice.